latest_migrations.manifest

```diff
@@ -3,6 +3,6 @@ auth: 0012_alter_user_first_name_max_length
 axes: 0006_remove_accesslog_trusted
 contenttypes: 0002_remove_content_type_name
 ee: 0014_roles_memberships_and_resource_access
-posthog: 0300_add_constraints_to_person_override
+posthog: 0301_auto_20230220_2333
```

Comment:
we should name migrations to help the future traveller

e.g. `DEBUG=1 ./manage.py makemigrations --name=record_annotations_are_automatic`

(I think the safest way is to delete and recreate the migration. Although you can do it manually)